align/schema/checker.py

```diff
@@ -182,7 +182,7 @@ def solve(self):
         r = self._solver.check()
         if r == z3.unsat:
             z3.set_option(max_depth=10000, max_args=100, max_lines=10000)
-            logger.debug(f"Unsat encountered: {self._solver}")
+            logger.info(f"Unsat encountered: {self._solver}")
```

Comment:
Is there a particular reason to make this info? This may reveal compiler's attempts to add constraints and confuse user.